include/kremlin/internal/types.h

```diff
@@ -72,13 +75,13 @@ typedef FStar_UInt128_uint128 FStar_UInt128_t, uint128_t;
 #include "kremlin/lowstar_endianness.h"
 
 /* This one is always included, because it defines C.Endianness functions too. */
-#if !defined(_MSC_VER)
+#if !defined(_MSC_VER) || defined(__clang__)
 #include "fstar_uint128_gcc64.h"
 #endif
 
-#if !defined(KRML_VERIFIED_UINT128) && defined(_MSC_VER)
+#if !defined(KRML_VERIFIED_UINT128) && defined(_MSC_VER) && !defined(__clang__)
 #include "fstar_uint128_msvc.h"
-#elif defined(KRML_VERIFIED_UINT128)
+#elif defined(KRML_VERIFIED_UINT128) && !defined(__clang__)
```

Comment:
I think this one shouldn't be here. 

You still want to enable clang64/windows and verified uint128. This line would make this scenario impossible (I believe).